akka-actor/src/main/scala/akka/actor/FSM.scala

```diff
@@ -238,7 +238,7 @@ object FSM {
  *     startWith(One, Data(42))
  *     when(One) {
  *         case Event(SomeMsg, Data(x)) => ...
- *         case Ev(SomeMsg) => ... // convenience when data not needed
+ *         case Event(SomeMsg, _) => ... // when data not needed
```

Comment:
Somewhat confusing, since the above `case Event(SomeMsg, Data(x))` will probably match before this. Perhaps use another msg, `SomeOtherMsg`.